core/primitives/src/runtime/parameter_table.rs

```diff
@@ -1,23 +1,57 @@
 use super::config::{AccountCreationConfig, RuntimeConfig};
+use near_primitives_core::account::id::ParseAccountError;
 use near_primitives_core::config::{ExtCostsConfig, VMConfig};
 use near_primitives_core::parameter::{FeeParameter, Parameter};
 use near_primitives_core::runtime::fees::{RuntimeFeesConfig, StorageUsageConfig};
+use near_primitives_core::types::AccountId;
 use num_rational::Rational;
-use serde::de::DeserializeOwned;
-use std::any::Any;
 use std::collections::BTreeMap;
 
+/// Represents values supported by parameter config.
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug, PartialEq)]
+#[serde(untagged)]
+pub(crate) enum ParameterValue {
+    Null,
+    U64(u64),
+    Rational { numerator: isize, denominator: isize },
+    String(String),
+}
+
+impl ParameterValue {
+    fn get_u64(&self) -> Option<u64> {
```

Comment:
Optional nit: The naming I have seen the most for such down-casting getters would be `as_u64`. That would be consistent with [serde_json::Value](https://docs.rs/serde_json/latest/serde_json/value/enum.Value.html).

But `.get_u64()` is also fine, mostly just wanted to make you aware of existing conventions.

Reply:
This is super useful to know! Changed the names to fit the convention.